intro/summary-exercises/optimize-fit.rst

```diff
@@ -56,8 +56,14 @@ and visualize it::
 
     >>> import matplotlib.pyplot as plt
     >>> t = np.arange(len(waveform_1))
-    >>> plt.plot(t, waveform_1) #doctest: +ELLIPSIS
+
+    >>> fig, ax = plt.subplots(figsize=(8, 6))
+    >>> ax.plot(t, waveform_1) #doctest: +ELLIPSIS
     [<matplotlib.lines.Line2D object at ...>]
+    >>> ax.set_xlabel('Time [ns]') #doctest: +ELLIPSIS
+    Text(0.5,0,'Time [ns]')
+    >>> ax.set_ylabel('Intensity [bins]') #doctest: +ELLIPSIS
+    Text(0,0.5,'Intensity [bins]')
```

Comment:
I wasn't sure whether `#doctest ELLIPSIS ` should have been included, but my code didn't pass the tests without it.